docker/Dockerfile

```diff
@@ -42,8 +42,11 @@ RUN apt-get update -y && \
   apt-get install curl ca-certificates gnupg software-properties-common -y && \
   curl -fsSL https://postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /usr/share/keyrings/postgresql-archive-keyring.gpg > /dev/null && \
   echo deb [arch=amd64,arm64,ppc64e1 signed-by=/usr/share/keyrings/postgresql.gpg] http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main | tee /etc/ap && \
+  curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc && \
```

Comment:
Docker in Docker :exploding_head: :smile:

Reply:
it's not really docker in docker since we use the same socket, its docker inisde but outside docker 🧌